Package.json

```diff
@@ -17,8 +17,8 @@
     "gulp-plumber": "1.1.0",
     "gulp-print": "2.0.1",
     "gulp-rename": "1.2.2",
-    "gulp-sass": "2.3.1",
-    "gulp-sourcemaps": "1.6.0",
+    "gulp-sass": "3.1.0",
+    "gulp-sourcemaps": "2.6.0",
     "gulp-typescript": "2.13.4",
```

Comment:
Wanna update this too https://www.npmjs.com/package/gulp-typescript ?? we are using quite an outdated version